src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const ManOrWomen =
+    sex === 'm' ? `${partnerName} is my wife` : `${partnerName} is my husband`;
```

Comment:
Please, before pushing check you naming, check how you can improve readability
here some links you can read about it
https://www.syncfusion.com/blogs/post/top-javascript-naming-convention
https://medium.com/@kjschelling/a-comprehensive-guide-to-naming-conventions-in-javascript-46a7518e4807

```suggestion
  const partnerLabel =`${partnerName} is my ${ sex === 'm' ? "wife" : "husband"}`;
```